runtime/near-vm-runner/fuzz/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "near-vm-runner-fuzz"
+version = "0.0.0"
+authors = ["Automatically generated"]
```

Comment:
Is it?

Reply:
It is. Though as `authors` field is now optional, it's better to remove it here (and upstream in the generator itself as well, cc @nagisa ).